app/redux/actions.js

```diff
@@ -44,3 +44,11 @@ export function stopCpuProfiling() {
     }
   };
 }
+
+export function updateCPUProfilingTable(cpuProfilingTable) {
+  return {
+    type: 'UPDATE_CPU_PROFILING_TABLE',
```

Comment:
Once the 'Stop CPU profiling' is clicked, the dump is available to the displayCPUProfile component. It takes this dump, processes the data, and puts it in the store under `cpuProfilingTable`.

Interactions like expanding or contracting the table keep updating the `cpuProfilingTable`. Thats the variable in the store that gets updated. Usually, data is added to the table.